langkit/compile_context.py

```diff
@@ -2124,6 +2124,10 @@ def dependencies(typ):
             # Filter types that are relevant for dependency analysis
```

Comment:
After I’ve read this commit, I failed to understand what it really does. What is shared exactly?